democracy_club/templates/home.html

```diff
@@ -26,8 +26,7 @@ <h2>Our impact in numbers</h2>
                 and local elections.</p>
 
             <ul class="ds-switcher ds-showcase" style="--gridCellMin: 16ch; --gridGap: 1.5rem;">
-                <li><strong>219m</strong> reminders to vote sent via partnerships in&nbsp;2019</li>
-                <li><strong>3.5m</strong> people used our services in the last general election</li>
+                <li><strong>14m</strong> people used our services in the 2024 general election</li>
                 <li><strong>83%</strong> of users say that our services make them more likely to&nbsp;vote</li>
```

Comment:
@pmk01 Regarding "more likely to vote", with the addition of the "no change" option, this number has become more complicated to report. Do we want to revert to 'found useful'? If I average the numbers from WDIV and WCIVF, we are at 86%.

Reply:
Yes, I agree, move back to 'found useful'.

Reply:
The specific question is 'did you find what you were looking for?' Is there's a better way to summarise this?
"85% of users found the information they needed", or similar?

Reply:
I've made this change.